src/client_side.h

```diff
@@ -492,4 +492,3 @@ std::ostream &operator <<(std::ostream &os, const ConnStateData::PinnedIdleConte
 std::ostream &operator <<(std::ostream &, const ConnStateData::ServerConnectionContext &);
 
 #endif /* SQUID_CLIENTSIDE_H */
-
```

Comment:
When the dust settles, please format changed sources. Re-adding incorrectly removed empty lines like this one are just some of the formatting changes that should probably be done before officially merging this branch.

Reply:
I think this comment is old, so far I couldn't see any large formatting changes

Reply:
This change request is old but still valid.